### PR TITLE
update home page text

### PR DESCRIPTION
**Pull request by KellyAH**
add extra info about who we are to the home page text.

changes

### CODE REVIEW

**Review by anaulin — APPROVED** [review]
Eggcellent! 🍳

File: src/index.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ layout: layouts/default.njk
 ---
 # The Zinc Collective
 
+🤗 We are an open-source volunteer coding group of diverse individuals. 
+
+🗓️ We meet weekly to work on coding projects.
```
Comment (zspencer) [inline src/index.md:8]: Very nice!

**Review by zspencer — APPROVED** [review]
🚢 :it:

**Comment by KellyAH** [comment]
🚀 deployed and ✅ verified on prod 💃

**Pull request merged**